lib/modules/dosomething/dosomething_campaign/includes/CampaignTransformer.php

```diff
@@ -83,7 +83,7 @@ public function show($id) {
    * @return array
    */
   protected function transform($item) {
-    $data = array();
+    $data = [];
```

Comment:
@DeeZone the only place we're still keeping `array()` syntax is in contributed Drupal modules we maintain.